tools/topology/topology2/include/pipelines/cavs/passthrough-playback.conf

```diff
@@ -62,7 +62,7 @@ Class.Pipeline."passthrough-playback" {
 				in_valid_bit_depth	24
 				out_bit_depth		32
 				out_valid_bit_depth	32
-				dma_buffer_size "$[$obs * 2]"
+				dma_buffer_size "$[$ibs * 2]"
```

Comment:
and oh I forgot t mention, we need to remove dma_buffer_size altogether. This is no longer used by the kernel and it computed based on ibs/obs and stream direction